lib/ansible/modules/windows/win_dns_client.py

```diff
@@ -19,15 +19,14 @@
   adapter_names:
     description:
       - Adapter name or list of adapter names for which to manage DNS settings ('*' is supported as a wildcard value).
-      - The adapter name used is the connection caption in the Network Control Panel or via C(Get-NetAdapter), eg C(Local Area Connection).
-    type: str
+      - The adapter name used is the connection caption in the Network Control Panel or the InterfaceAlias of C(Get-DnsClientServerAddress), eg C(Local Area Connection).
     required: yes
-  ipv4_addresses:
+  dns_servers:
     description:
-      - Single or ordered list of DNS server IPv4 addresses to configure for lookup. An empty list will configure the adapter to use the
+      - Single or ordered list of DNS servers (IPv4 and IPv6 addresses) to configure for lookup. An empty list will configure the adapter to use the
         DHCP-assigned values on connections where DHCP is enabled, or disable DNS lookup on statically-configured connections.
-    type: str
```

Comment:
Why remove the type?

Reply:
Also a list is allowed and only one type can be specified. Is there something better?

Reply:
set the type to list then

Reply:
but also string is accepted. How do I specify both?

Reply:
since a single string or a list in the firm of string are a list themselves I think a list type is OK